install-devstack-xen.sh

```diff
@@ -444,89 +445,55 @@ VM_BRIDGE_OR_NET_NAME="osvmnet"
 CINDER_SECURE_DELETE=False
 
 # Compute settings
-EXTRA_OPTS=("xenapi_disable_agent=True")
-API_RATE_LIMIT=False
 VIRT_DRIVER=xenserver
 
-# Use a XenServer Image
-#DOWNLOAD_DEFAULT_IMAGES=False
-#IMAGE_URLS="http://ca.downloads.xensource.com/OpenStack/cirros-0.3.4-x86_64-disk.vhd.tgz"
-#DEFAULT_IMAGE_NAME="cirros-0.3.4-x86_64-disk"
-
 # OpenStack VM settings
-OSDOMU_VDI_GB=40
-
-# Exercise settings
-ACTIVE_TIMEOUT=500
-TERMINATE_TIMEOUT=500
+OSDOMU_VDI_GB=30
+OSDOMU_MEM_MB=8192
 
-# Increase boot timeout for neutron tests:
-BOOT_TIMEOUT=500
+TERMINATE_TIMEOUT=90
+BUILD_TIMEOUT=600
 
 # DevStack settings
 LOGFILE=${SCREEN_LOGDIR}/stack.log
 SCREEN_LOGDIR=${SCREEN_LOGDIR}
 
+UBUNTU_INST_HTTP_HOSTNAME=apt-cacher.xenrt.citrite.net
+UBUNTU_INST_HTTP_DIRECTORY=/ubuntu
+
 # Turn on verbosity (password input does not work otherwise)
 VERBOSE=True
 
 # XenAPI specific
 XENAPI_CONNECTION_URL="http://$XENSERVER_IP"
 VNCSERVER_PROXYCLIENT_ADDRESS="$XENSERVER_IP"
 
-MULTI_HOST=1
+# Neutron specific part
+ENABLED_SERVICES+=neutron,q-domua
+Q_ML2_PLUGIN_MECHANISM_DRIVERS=openvswitch
 
-# Skip boot from volume exercise
-# See https://bugs.launchpad.net/openstack-ci/+bug/1263824 for euca and bundle
-SKIP_EXERCISES="boot_from_volume,bundle,euca"
+Q_ML2_PLUGIN_TYPE_DRIVERS=vlan,flat
+ENABLE_TENANT_TUNNELS=False
+ENABLE_TENANT_VLANS=True
+Q_ML2_TENANT_NETWORK_TYPE=vlan
+ML2_VLAN_RANGES="physnet1:1100:1200"
 
-ENABLED_SERVICES=g-api,g-reg,key,n-api,n-crt,n-cpu,n-sch,horizon,mysql,rabbit,sysstat,tempest,s-proxy,s-account,s-container,s-object,cinder,c-api,c-vol,c-sch,n-cond,heat,h-api,h-api-cfn,h-api-cw,h-eng,n-net,n-novnc,n-cauth
+PUB_IP=172.24.4.1
+SUBNETPOOL_PREFIX_V4=192.168.10.0/24
+NETWORK_GATEWAY=192.168.10.1
 
-# Set instance build timeout to 300s in tempest.conf
-BUILD_TIMEOUT=300
-
-# XEN_FIREWALL_DRIVER=nova.virt.xenapi.firewall.Dom0IptablesFirewallDriver
-XEN_FIREWALL_DRIVER=nova.virt.firewall.NoopFirewallDriver
-
-# 9 Gigabyte for object store
-SWIFT_LOOPBACK_DISK_SIZE=9663676416
-
-# Additional Localrc parameters here
-INSTALL_TESTONLY_PACKAGES=True
-
-UBUNTU_INST_HTTP_HOSTNAME=us.archive.ubuntu.com
-UBUNTU_INST_HTTP_DIRECTORY=/ubuntu
-
-LOCALRC_CONTENT_ENDS_HERE
-
-cat << LOCALCONF_CONTENT_ENDS_HERE > local.conf
-# ``local.conf`` is a user-maintained settings file that is sourced from ``stackrc``.
-# This gives it the ability to override any variables set in ``stackrc``.
-# Also, most of the settings in ``stack.sh`` are written to only be set if no
-# value has already been set; this lets ``local.conf`` effectively override the
-# default values.
-
-# The ``localrc`` section replaces the old ``localrc`` configuration file.
-# Note that if ``localrc`` is present it will be used in favor of this section.
-# --------------------------------
-[[local|localrc]]
+VLAN_INTERFACE=eth1
+PUBLIC_INTERFACE=eth2
 
 # Nova user specific configuration
 # --------------------------------
-
-[[post-config|$NOVA_CONF]]
+[[post-config|/etc/nova/nova.conf]]
```

Comment:
Can't we use devstack's variable names here?
[[post-config|\\$NOVA_CONF]]

Agreed if you use $NOVA_CONF it will use the var from bash but see http://docs.openstack.org/developer/devstack/configuration.html